scp.c

```diff
@@ -1262,8 +1262,9 @@ source(int argc, char **argv)
 	off_t i, statbytes;
 	size_t amt, nr;
 	int fd = -1, haderr, indx;
-	char *last, *name, buf[PATH_MAX + 128], encname[PATH_MAX];
+	char *last, *name, *buf, *encname;
```

Comment:
Please add the github issue link to description.

Have #ifdef WINDOWS block as this is a common code. Add some comments why we are deviating from original implementation.